task8/{UPMessenger}/src/main/java/Tweets.java

```diff
@@ -0,0 +1,61 @@
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.BufferedReader;
+import java.io.IOException;
+
+import java.util.*;
+
+
+@WebServlet("/tweets/*")
+public class Tweets extends HttpServlet {
+    private void getPost(String id, HttpServletResponse response) throws IOException {
```

Comment:
Private methods go below public. Methods that are called from inside other methods, go after the methods that do calling. this allows to read code as text - from top to bottom -- not scrolling back-and-forth

Reply:
Also, pleade do never name a `void` method like `get...` This way getters are names = the methods that return something. so this is confusing